app/app.go

```diff
@@ -147,7 +147,7 @@ const (
 	Name                 = "elys"
 
 	// Temperal declaration until we have dex module
-	DexRevenueCollectorName = "dev_revenue_collector"
+	DexRevenueCollectorName = "dexRevenueCollector"
```

Comment:
I think this name could be kept as "dev_revenue_collector" or would be good to update to "DexRevenueCollector" - since it's not a variable name.